lib/dialog.rb

```diff
@@ -1,7 +1,6 @@
 class Dialog
 
-	def initialize(game_state, args = {})
-		@game_state = game_state
+	def initialize(args = {})
 		@output = args[:output] ||= ->(string) { puts string }
 		@output_2 = args[:output_2] ||= ->(string) { print string }
```

Comment:
output_2....maybe output_alternative? This doesn't feel like a descriptive name quite yet